examples/3box/.mm-plugin.json

```diff
@@ -0,0 +1,3 @@
+{
+  "port": 8081
```

Comment:
We're going to incorporate this as a default, so it would be helpful if you could change all references to port `8081` (which is used for another example) to `8087`.

Reply:
Also, please rename `.mm-plugin.json` to `mm-plugin.config.json` (the former is still available for backwards-compatibility, but the latter is the new standard).